test/e2e/specs/site-editor/user-global-styles-revisions.spec.js

```diff
@@ -148,9 +148,6 @@ class UserGlobalStylesRevisions {
 	}
 
 	async openRevisions() {
-		await this.page
-			.getByRole( 'menubar', { name: 'Styles actions' } )
-			.click();
 		await this.page.getByRole( 'button', { name: 'Revisions' } ).click();
```

Comment:
Not something that should block this PR but I think we can now just inline this action instead of calling `openRevisions()`.

Reply:
Oh for sure, I'll update the PR. 👍🏻 Thanks